Backpack/Calendar/Classes/BPKCalendarCell.m

```diff
@@ -85,57 +85,65 @@ - (void)layoutSubviews {
 
     self.shapeLayer.frame = CGRectMake(paddingX, 0, BPKCalendarCellSpacing.cellCircleHeight, BPKCalendarCellSpacing.cellCircleHeight);
 
+    CGFloat underflow = 0;
+    CGFloat overflow = 0;
+    CGFloat overflowLength = [[UIScreen mainScreen] bounds].size.width;
```

Comment:
This needlessly limits the view to being full screen. I think we need to think of another method for this

Reply:
For example, we could keep a `weak` reference to the calendar itself and use that instead

Reply:
I don't think it does? It just ensures that the overflow will be long enough regardless of screen size?

Reply:
It feels wrong 🤔 If this is the case why not use 9000?

Reply:
That would work too?
Until apple release a device that's wider than 9000pt

Reply:
I feel like it should be the exact value required calculated, but maybe that's just dumb

Reply:
Can we add a comment to overflowLength that explains why that values is used?